sunpy/map/mapcube.py

```diff
@@ -178,16 +178,16 @@ def annotate_frame(i):
             axes.set_title("{s.name}".format(s=self[i]))
 
             # x-axis label
-            if self[0].coordinate_system.x == 'HG':
-                xlabel = 'Longitude [{lon}'.format(lon=self[i].spatial_units.x)
+            if self[0].coordinate_system.axis1 == 'HG':
```

Comment:
This one is on me, I must have missed this.

It's better to replace this whole block with this from map.plot: https://github.com/sunpy/sunpy/blob/master/sunpy/map/mapbase.py#L1719

As this label-gen code was used in a few places I moved it into a helper function in visualization.